src/app/auth/login/login.component.ts

```diff
@@ -26,22 +28,33 @@ export class LoginComponent {
     
       });
     }
-    
-    login() {
-  
-      this.usuarioService.login(this.loginForm.value).subscribe(resp=>{ 
-        // Navegar al Dashboard
-        this.router.navigateByUrl('/');
-        console.log(resp)
-        
-        }, (err)=> {
-          console.log(err);
-          this.loginIncorrecto = true
-          Swal.fire('Error', err.error.msg, 'error');
-        });
-  
   
+    get emailRequerido(){
+      return this.loginForm.get('email').invalid && this.loginForm.get('email').touched
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138940)
```suggestion
      return this.loginForm.get("email").invalid && this.loginForm.get('email').touched
```